src/api/app/views/webui/projects/maintenance_incidents/_incident_table_entries.html.erb

```diff
@@ -1,16 +1,18 @@
 <% incidents.each do |incident| %>
     <% index = 0 %>
-    <% release_targets_ng = incident.release_targets_ng() %>
+    <% release_targets = incident.target_repositories.joins(:project).
```

Comment:
Why do you need this N+1 query here? `target_repositories` should be prefetched for the full `incidents` array by the controller before going into the view.